qiskit/transpiler/passes/basis/decompose.py

```diff
@@ -30,20 +30,23 @@ def __init__(
         self,
         gate: Optional[Type[Gate]] = None,
         gates_to_decompose: Optional[Union[Type[Gate], List[Type[Gate]], List[str], str]] = None,
+        reps: int = 1,
```

Comment:
I would remove this again - `reps` as an argument to a pass doesn't make too much sense, because you can always do `for _ in [None] * reps: pass_.run(dag)`, which is clearer at the point of use.